framework/source/class/qx/ui/basic/Image.js

```diff
@@ -780,7 +876,36 @@ qx.Class.define("qx.ui.basic.Image",
      * @param source {String} source path
      */
     __setSource: function (el, source) {
-      if (el.getNodeName() == "div") {
+      var isFont = source && qx.lang.String.startsWith(source, "@");
+
+      if (isFont) {
+        var ResourceManager = qx.util.ResourceManager.getInstance();
+        var font = qx.theme.manager.Font.getInstance().resolve(source.match(/@([^/]+)/)[1]);
+        el.setStyles(font.getStyles());
```

Comment:
This might override styles already set by the Appearance (e.g. color), so it needs some checks here to prevent such behaviour although I currently do not know what would be the best strategy here.